src/app.ts

```diff
@@ -5,6 +5,9 @@ import * as path from "path";
 import * as config from "./config";
 import logger from "./utils/logger";
 import router from "./routes";
+import { CsrfProtectionMiddleware } from "@companieshouse/web-security-node";
```

Comment:
it is just a small detail, but is it possible to keep the library imports at the top and then the local imports?

Reply:
Ordered imports